docs/hello-world.md

```diff
@@ -4,13 +4,11 @@ title: Hello World!
 sidebar_label: Hello World!
 ---
 
-[Examples](https://github.com/neon-bindings/examples)
+**Full source code:** [examples/cpu-count](https://github.com/neon-bindings/examples/tree/main/examples/cpu-count)
 
-This guide will walk you through writing, building, and running your first Neon project. We'll try to walk you through each step carefully, but if you want to skip ahead, you can always go straight to the [full demo](https://github.com/neon-bindings/examples/tree/main/examples/cpu-count) in the examples repository.
+This simple example project will be a tiny module that returns the number of processors in the current device. If you're not familiar with fancy systems concepts like processors and CPUs, don't panic! We'll be using the [`num_cpus`](https://crates.io/crates/num_cpus) crate to do all the heavy lifting for us, and we'll just return the number it gives us.
```

Comment:
The word "simple" is used a lot. Can I suggest trying to find synonyms? I.e., "small example" instead of "simple." At least to me, "simple" is a subjective term and may or may not apply depending on the readers previous experience.

Reply:
Great point, "simple" can be intimidating and non-inclusive.